Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Note this image doesn't really matter for hostprocess
```

Comment:
I think the biggest issue w/ using a mis-matched image for host-process containers is added overhead from needing to pull the nanoserver base layers for a different OS version.
This will work but long-term maybe we should at least have a ltsc2022 image too.

Reply:
added logic to build both 2019 and 2022